tools/run_tests/dockerize/build_and_run_docker.sh

```diff
@@ -216,7 +216,10 @@ fi
 # Copy contents of OUTPUT_DIR back under the git repo root
 if [ "${OUTPUT_DIR}" != "" ]
 then
-  cp -r "${TEMP_OUTPUT_DIR}/${OUTPUT_DIR}" "${git_root}" || DOCKER_EXIT_CODE=$?
+  # use -f to copy existing *tar.gz and *py3-none-any.whl files
+  # without throwing 'file exists' error
+  cp -rf "${TEMP_OUTPUT_DIR}/${OUTPUT_DIR}" \
```

Comment:
Why do we need this? I'd like to understand the reason we're running into this problem.